docs/plugins/find_and_install_plugin.md

```diff
@@ -30,3 +30,13 @@ directly from within napari:
    ![napari viewer's Plugin dialog. At the bottom of the dialog, there is a place to install by name, URL, or dropping in a file.](/images/plugin-install-dialog.png)
 
 3. Click the “Install” button next to the input bar.
+
+```{note}
+When wanting to install a plugin from a URL you need to respect a specific syntax required by pypi!
+You need to add the vcs (version control system) information before the URL for pip to be able to handle the URL.
+You will find more information about this on the [pip documentation](https://pip.pypa.io/en/stable/topics/vcs-support/)
```

Comment:
```suggestion
Napari supports installing plugins from VCS (ex. `git`, `svn`) by passing the URL to the repository in the search bar. As Napari follows the `pip` way more information about URL format may be found in [`pip` documentation](https://pip.pypa.io/en/stable/topics/vcs-support/)
```

Maybe something like this. There is a try to reduce technical details and highlight that this is only about installation from VCS, not any URL. 

This is my only suggestion, as my writing skills are not excellent.